SU-HSDO/suhumsci
# feat(STN-368):  tighten spacing for traditional theme cards

# READY FOR REVIEW

## Summary
Paired with J to do design implementation: tightens up spacing on all card patterns. Adds a new 'xxxs' spacing unit to do so.

- [x] ensure padding on colorful still looks right

## Steps to Test
1. Ensure that `npm run test` passes in humsc_basic
2. Check over the following areas for changes:

**Vertical Card**
<img width="336" alt="Screen Shot 2020-05-14 at 12 05 43 PM" src="https://user-images.githubusercontent.com/12848168/81959031-e117d280-95dc-11ea-8a53-bc351f12952b.png">
<img width="943" alt="Screen Shot 2020-05-14 at 12 05 47 PM" src="https://user-images.githubusercontent.com/12848168/81959044-e6751d00-95dc-11ea-8c59-0db87f28c06c.png">

**Horizontal Card**
<img width="980" alt="Screen Shot 2020-05-14 at 12 10 20 PM" src="https://user-images.githubusercontent.com/12848168/81959061-ec6afe00-95dc-11ea-9869-cdab1fb3285b.png">

**Structured Card**
<img width="953" alt="Screen Shot 2020-05-14 at 12 10 30 PM" src="https://user-images.githubusercontent.com/12848168/81959073-f12fb200-95dc-11ea-94ec-7d42a133dc95.png">

**Vertical Linked Card**
<img width="968" alt="Screen Shot 2020-05-14 at 12 10 06 PM" src="https://user-images.githubusercontent.com/12848168/81959129-00166480-95dd-11ea-9e22-0cfaabcb225f.png">

**Date Stacked Vertical Card**
<img width="310" alt="Screen Shot 2020-05-14 at 12 09 23 PM" src="https://user-images.githubusercontent.com/12848168/81959190-17555200-95dd-11ea-843e-4374371b4128.png">

**Date Stacked Horizontal Card**
<img width="945" alt="Screen Shot 2020-05-14 at 12 09 43 PM" src="https://user-images.githubusercontent.com/12848168/81959224-20462380-95dd-11ea-99b6-effa399a0ce0.png">

**Post Cards**
<img width="997" alt="Screen Shot 2020-05-14 at 12 10 00 PM" src="https://user-images.githubusercontent.com/12848168/81959265-2c31e580-95dd-11ea-92fa-643cf7c86323.png">

**All of these in a grid.**



## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
